x-pack/plugins/security_solution/public/management/pages/policy/index.tsx

```diff
@@ -6,15 +6,32 @@
  */
 
 import React, { memo } from 'react';
-import { Route, Switch } from 'react-router-dom';
+import { Route, Switch, Redirect } from 'react-router-dom';
 import { PolicyDetails } from './view';
-import { MANAGEMENT_ROUTING_POLICY_DETAILS_PATH } from '../../common/constants';
+import {
+  MANAGEMENT_ROUTING_POLICY_DETAILS_FORM_PATH,
+  MANAGEMENT_ROUTING_POLICY_DETAILS_TRUSTED_APPS_PATH,
+  MANAGEMENT_ROUTING_POLICY_DETAILS_PATH_OLD,
+} from '../../common/constants';
 import { NotFoundPage } from '../../../app/404';
+import { getPolicyDetailPath } from '../../common/routing';
 
 export const PolicyContainer = memo(() => {
   return (
     <Switch>
-      <Route path={MANAGEMENT_ROUTING_POLICY_DETAILS_PATH} exact component={PolicyDetails} />
+      <Route
+        path={[
```

Comment:
Did not realize that `path` prop could take an array 👍

Reply:
https://reactrouter.com/web/api/Route/path-string-string